test/unit_Polytope.jl

```diff
@@ -229,11 +229,8 @@ if test_suite_polyhedra
         p2 = VPolytope([v3, v4])
         ch = convex_hull(p1, p2)
         vl = vertices_list(ch)
+        @test length(vl) == 4
         @test v1 ∈ vl && v2 ∈ vl && v3 ∈ vl && v4 ∈ vl
```

Comment:
These two lines can now be made more readable with `ispermutation(vl, [v1, v2, v3, v4])`.